Bowling.java

```diff
@@ -2,25 +2,91 @@
 
 public class Bowling {
 
-  private int totalPoints;
+  private Scoreboard scoreboard;
 
   public Bowling() {
-    totalPoints = 0; 
+    scoreboard = new Scoreboard();
+    scoreboard.addFrame();
   }
 
-  public Bowling bowl(int pinsDown) {
-		if (pinsDown < 0) {
-			throw new IllegalArgumentException("Cannot knock down negative pins. Bowl again.");
-		}
-    totalPoints += pinsDown;
+  public Bowling bowl(Integer pinsDown) {
+    if (pinsDown < 0) {
+      throw new IllegalArgumentException("Cannot knock down negative pins. Bowl again.");
+    }
+
+    Frame currentFrame = scoreboard.getCurrentFrame();
```

Comment:
Maybe Bowling should just be aware of Frames and we don't have a Scoreboard object?